File: android/OhDormitory/app/src/main/java/kr/hs/emirim/uuuuri/ohdormitory/Model/CleanNotice.java
package kr.hs.emirim.uuuuri.ohdormitory.Model;

import java.util.HashMap;
import java.util.Map;

public class CleanNotice extends Notice {
    private Map<Integer, Integer> cleanList;

    public CleanNotice(int notice_id, String title, String w_time, int type, Map<Integer, Integer> cleanList) {
        super(notice_id, title, w_time, type);
        this.cleanList = cleanList;
    }

    public Map<Integer, Integer> getCleanList() {
        return cleanList;
    }
}

File: ios/OhDormitory/OhDormitory/CleanNotice.swift
//
//  CleanNotice.swift
//  OhDormitory
//
//  Created by 장명수 on 2018. 6. 11..
//  Copyright © 2018년 doori. All rights reserved.
//

import Foundation

class CleanNotice : Notice{
    var detail : [String]
    init(notice_id:Int, title:String, type:Int, w_time:String, d_time:String, detail:[String]){
        self.detail = detail
        super.init(notice_id: notice_id, title: title, type: type, w_time: w_time, d_time: d_time)
    }
}
